go/test/endtoend/onlineddl/scheduler/onlineddl_scheduler_test.go

```diff
@@ -362,6 +362,13 @@ func testScheduler(t *testing.T) {
 		`
 	)
 
+	testCompletedTimetamp := func(t *testing.T, uuid string) {
+		rs := onlineddl.ReadMigrations(t, &vtParams, uuid)
+		require.NotNil(t, rs)
+		for _, row := range rs.Named().Rows {
+			assert.False(t, row["completed_timestamp"].IsNull())
```

Comment:
Can we also assert/confirm that the timestamp is a valid timestamp (e.g. not all 0's) and is recent (within the last hour e.g.)? I think that more accurately tests/confirms what we want to. 

Would it also make sense to do this in a defer for all of them and only do the assert if the status is Completed?

Reply:
Good idea and done. Now validating correctness of `completed_timestamp` for all migrations in `onlineddl_vrepl_test` and in `onlineddl_scheduler_test` at the end of the suite run.Validating that the value is parseable and that it's greater than tests start up time.